api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -266,7 +266,7 @@ static ServiceContext getServiceContext() {
 	 *
 	 * @param ctx
 	 */
-	public void setServiceContext(ServiceContext ctx) {
+	public void synchronized setServiceContext(ServiceContext ctx) {
```

Comment:
synchronized should be before void. Seems you didn't run `mvn clean package` after making this change else the compiler would have caught it. See travis error here https://travis-ci.org/openmrs/openmrs-core/builds/320193916#L2148

Reply:
I saw the error but didn't understand it. I thought it had to do with installation of plugins, and incorrectly ignored it. I have corrected my commit.